router/router.go

```diff
@@ -666,7 +666,7 @@ func (worker *workerT) processDestinationJobs() {
 							)
 
 							stats.NewTaggedStat("transformer_proxy.output_events_count", stats.CountType, stats.Tags{
-								"destType":      worker.rt.destName,
+								"destType":      strings.ToLower(worker.rt.destName),
```

Comment:
why are we using lowercase?

Reply:
Basically the destination names are lower-cased in transformer, since we are sending a lower-cased value for `destType` tag for the stats send from transformer.
When we setup any alerts we need to group using `destType` which needs to in-sync with what we send from transformer

Reply:
Not related to this PR, but we need to ensure that destType is represented in the same form across all of our services. It is used as an identifier in many instances, and having it in mixed cases will cause problems. My suggestion would be that transformer is changed to work with normal-casing, but not sure regarding the effort behind this.